runtime/doc/starting.txt

```diff
@@ -1386,9 +1386,32 @@ STATE DIRECTORY (DEFAULT) ~
     Unix:         ~/.local/state              ~/.local/state/nvim
     Windows:      ~/AppData/Local             ~/AppData/Local/nvim-data
 
+CACHE DIRECTORY (DEFAULT) ~
+                  *$XDG_CACHE_HOME*             Nvim: stdpath("cache")
+    Unix:         ~/.cache	              ~/.cache/nvim
+    Windows:      ~/AppData/Local/Temp        ~/AppData/Local/Temp/nvim-data
+
+LOG FILE (DEFAULT) ~
+                  *$NVIM_LOG_FILE*	      Nvim: stdpath("log")
+    Unix:         ~/.local/state/nvim	      ~/.local/state/nvim/log
+    Windows:      ~/AppData/Local/nvim-data   ~/AppData/Local/nvim-data/log
+
+ADDITIONAL CONFIGS DIRECTORY (DEFAULT) ~
+                  *$XDG_CONFIG_DIRS*            Nvim: stdpath("config_dirs")
+    Unix:         /etc/xdg/		      /etc/xdg/nvim
+    Windows:      Not applicable	      Not applicable
+
+ADDITIONAL DATA DIRECTORY (DEFAULT) ~
+                  *$XDG_DATA_DIRS*              Nvim: stdpath("data_dirs")
+    Unix:         /usr/local/share	      /usr/local/share/nvim
+		  /usr/share                  /usr/share/nvim
+    Windows:      Not applicable	      Not applicable
+
```

Comment:
The entries above use spaces for alignment, so please align using spaces instead of TABs here.